tests/integration/gh-6970/src/test/resources/application.yaml

```diff
@@ -15,6 +15,15 @@
 #
 
 tracing:
-  service: "helidon-service"
-  protocol: "https"
-  host: "jaeger"
\ No newline at end of file
+  enabled: true
+  global: true
+  service: "example"
+  log-spans: false
+  protocol: "http"
+  host: "localhost"
+  port: 14250
+  path: "/api/traces"
+  propagation: "w3c"
```

Comment:
If you specify only w3c will there be any traceId/spanId/parentSpanId/sampled values propagated? I know this single test doesn't rely on that (only baggage), but won't any other tests that are influenced by this yaml have the side-effect of not getting their traceId etc? Just trying to understand.